Hangman.py

```diff
@@ -0,0 +1,68 @@
+# This is a word game
+from random import choice
+from collections import Counter
+
+
+def selectWord(fname):
+    words = open(fname).readlines()
+    myword = choice(words)
+    myword = str(myword).strip('  \n')
+    myword = myword.lower()
+    return myword
+
+
+def LetterFrequency(text,size):

```

Comment:
According to PEP8: 

> "Function names should be lowercase, with words separated by underscores as necessary to improve readability."

It'd be better to name the function something like this: **calculate_letter_frequency(text, size)**